tests/testthat/test-aalen_johansen.R

```diff
@@ -0,0 +1,13 @@
+test_that("Aalen Johansen works as expected", {
```

Comment:
we need more tests, so that we cover all the 3 cases with c1/c2 values as seen in the function code